server/actions/Tree.ts

```diff
@@ -16,6 +16,29 @@ import { Tree } from "../../utils/types";
     await TreeSchema.create(tree);
 };
 
+/**
+ * @param queryTree tree object containing only ID of the 
+ * @returns a single tree
+ */
+export const getTree = async function(queryTree: Tree) {
+    await mongoDB();
+    
+    var keys = Object.keys(queryTree);
```

Comment:
Since this value isn't changed anywhere, you can make it `const` instead of `var`.